README.md

```diff
@@ -99,6 +99,26 @@ point. You can, however, change this behaviour by invoking such
 commands with a prefix argument. For instance: `C-u C-c C-v` will ask
 for the symbol you want to show the docstring for.
 
+#### *WARNING* - Standard REPLs
+
+Note that if you decide _NOT_ to use the socket repl, it is highly recommended
+you disable output coloring and/or readline facilities: `inf-clojure` does not
+filter out ASCII escape characters at the moment and will not behave correctly.
```

Comment:
The issue of speaking with the terminal is a big one. Comint hides some of the complexity but cannot eliminate it. Things like embedded `readline` and colors all change behaviors and I am so far from being an expert that I was even wondering whether console support should be dropped completely at some point (maybe by adopting another Emacs socket-only package). Sooner or later...